src/app/(nextjs_migration)/(authenticated)/user/layout.tsx

```diff
@@ -29,7 +28,7 @@ export type Props = {
 
 const MainLayout = async (props: Props) => {
   const session = await getServerSession(authOptions);
-  if (!session) {
+  if (!session || !session.user?.subscriber) {
```

Comment:
```suggestion
  if (!session?.user?.subscriber) {
```

Reply:
Also I think we can then remove some `?`s from line 35 :grin:

Reply:
Yes — updated in [7c024e2](https://github.com/mozilla/blurts-server/pull/3817/commits/7c024e2b4c203d3653fd472e33e0cc529106f67c).